grails-app/conf/application.yml

```diff
@@ -632,6 +632,84 @@ openboxes:
             enabled: true
             pageSize: 10
 
+#Add classes and event type and event closure for hibernate persistence handler.
+    publishEventForEntitiesAndFields:
+        User:
+            PRE_UPDATE:
+                eventName: onEventUserUpdated
+                fields:
+                    - ALL
+        Product:
+            INSERT:
+                eventName: onEventProductCreated
+                fields:
+                    - ALL
+            PRE_UPDATE:
+                eventName: onEventProductUpdated
+                fields:
+                    - ALL
+        Shipment:
+            INSERT:
+                eventName: onEventShipmentCreated
+                fields:
+                - ALL
+            PRE_UPDATE:
+                eventName: onEventShipmentUpdated
+                fields:
+                - currentStatus
+
+    application:
+        event:
+            webhook:
+                - hooks:
```

Comment:
`openboxes: 
    notifications:
        - provider: webhook
          enabled: true`
          
 This will work if only one webhook we need to send. but as you said we can configure multiple webhook also
 so how we will manage multiple webhooks here.